data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,57 @@ mission "Arachi Cartoonist 2"
+				goto end
+			label seed
+			`	Once you have the seed in your hands, the organizers tell you to go through the path on the left, and so you do.`
+			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it, while keeping your eyes on the small stage in front of you. After the audience is all here, the curtains open, to reveal many small figures made out of what looks like paper, in a style that loosely resembles origami.`
+			`	The central figure is a young Saryd boy, who is galloping in a dense forest, as the sound effects of thunder grow increasingly frequent. Lightning strikes one of the trees on the way, and it falls in flames, blocking the boy's path, so he turns to run elsewhere. Another lightning bolt, and another tree falls. After a few more repetitions, the boy is too tired to run anymore, hurt by some of the flames and now limping through the forest.`
+			`	Whimpering as the thunder grows closer once more, he spots a tree unlike the others, a great oak with branches thicker than the trees around him, so he goes to it. Laying down at the base of that largest tree, he falls asleep, and the curtain closes.`
+			`	It opens once again after a few moments, with the boy waking up in darkness. When he gets up, rumbling sounds are accompanie by rays of light reaching the ground, and he realizes the great oak had lowered its branches around him, and as it lifts them up once again, it knocks back several trees that had fallen on them during the storm. The sun as his guide, the boy manages to leave the forest, and the curtains close again.`
+				goto end
+			label pearl
+			`	Once you have the seashell in your hands, the organizers tell you to go through the path on the middle, and so you do.`
+			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it, while keeping your eyes on the small stage in front of you. After the audience is all here, the curtains open, to reveal many small figures made out of what looks like paper, in a style that loosely resembles origami.`
+			`	A Saryd is on a boat, fishing at sea. He feels a bite, but once he reels it in there is nothing on the hook. He feels another bite, and again finds nothing. Time and again, he looks to the hook to find the bait still there, intact, but continues to feel the bites, that are sometimes so blatant even the boat moves a little. The day moves by quickly, with a piece of paper representing the sun moving swiftly from one side of the "sky" to the other.`
```

Comment:
This is another place where <First> is unlikely to have the needed experience to grasp this instantly. For simple biomechanical reasons a Saryd would be unable to use a small boat in the human style. So if they have small individual boats they would look quite different indeed to our conception of a small fishing boat.
Personally I don't think Saryds would use individual boats, but if they're going to then some thought needs to be put into describing the boat, because a small Saryd boat probably wouldn't be recognisable to a human as being a boat without a lot of context that would be hard to express through origami.